src/lib/device-types/storage/s3.ts

```diff
@@ -12,24 +12,24 @@ import {
 export const getKey = (...parts: string[]): string => parts.join('/');
 
 class UnauthenticatedS3Facade {
-	constructor(private s3Client: AWS.S3) {}
+	constructor(private s3: AWS.S3) {}
```

Comment:
Nit: I personally prefer `client` over `s3`, as it more accurately describes it.

Reply:
I prefer s3 and went with it because `client` is pretty generic; it could refer to a pinejs client, a redis client, etc, whereas `s3` makes it clear what it talks to, eg `s3.makeUnauthenticatedRequest` is obvious where it's going whereas `client.makeUnauthenticatedRequest` is not. For this specific case the code is short enough that you can see all the context at once so it doesn't matter much either way, but if it was larger it might, particularly if we mixed multiple types of client.

That said I'd prefer `s3Client` but there was no easy refactoring to make the name available